js/src/shell/js.cpp

```diff
@@ -5328,44 +5356,84 @@ static bool Parse(JSContext* cx, unsigned argc, Value* vp) {
   ParseInfo parseInfo(cx, allocScope);
 
   RootedScriptSourceObject sourceObject(
-      cx, frontend::CreateScriptSourceObject(cx, options, Nothing()));
+                                        cx, frontend::CreateScriptSourceObject(cx, options, Nothing()));
```

Comment:
This indentation looks strange.
is this the result of clang-format?

Reply:
I am not using clang-format because for reasons which are still unknown to me, it keep trying to undo all the existing formatting, such as:

```diff
-bool OptionParser::addOptionalMultiStringArg(const char* name,
-                                             const char* help) {
+bool OptionParser::addOptionalMultiStringArg(const char *name,
+                                             const char *help) {
```